src/util/triplea/mapXmlCreator/CanalDefinitionsPanel.java

```diff
@@ -0,0 +1,450 @@
+package util.triplea.mapXmlCreator;
+
+import java.awt.Color;
+import java.awt.FontMetrics;
+import java.awt.Graphics;
+import java.awt.Graphics2D;
+import java.awt.Point;
+import java.awt.event.MouseEvent;
+import java.awt.geom.Rectangle2D;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.Optional;
+import java.util.Set;
+import java.util.SortedSet;
+import java.util.TreeSet;
+
+import javax.swing.JOptionPane;
+import javax.swing.SwingUtilities;
+
+import com.google.common.collect.Maps;
+import com.google.common.collect.Sets;
+
+import games.strategy.common.swing.SwingAction;
+import util.triplea.mapXmlCreator.TerritoryDefinitionDialog.DEFINITION;
+
+
+final public class CanalDefinitionsPanel extends ImageScrollPanePanel {
+  static final String HTML_CANAL_KEY_POSTFIX = ": ";
```

Comment:
Hi DanVanAtta,

sry for using this way again to responde, but I get the following
message on reviewable:
/(Reviewable was unable
<https://github.com/Reviewable/Reviewable/issues/14> to map this GitHub
line comment to the right spot—sorry!)/

This HTML strings should be summed up later in a constant class and used
while map XML file creation AND map XML file reading. I have added a
TODO comment.

Am 10.04.2016 um 23:55 schrieb Dan Van Atta:
>
> In src/util/triplea/mapXmlCreator/CanalDefinitionsPanel.java
> <https://github.com/triplea-game/triplea/pull/598#discussion_r59142323>:
>
> > +import java.util.Set;
> > +import java.util.SortedSet;
> > +import java.util.TreeSet;
> > +
> > +import javax.swing.JOptionPane;
> > +import javax.swing.SwingUtilities;
> > +
> > +import com.google.common.collect.Maps;
> > +import com.google.common.collect.Sets;
> > +
> > +import games.strategy.common.swing.SwingAction;
> > +import util.triplea.mapXmlCreator.TerritoryDefinitionDialog.DEFINITION;
> > +
> > +
> > +final public class CanalDefinitionsPanel extends ImageScrollPanePanel {
> > +  static final String HTML_CANAL_KEY_POSTFIX = ": ";
>
> Can these be made private?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/triplea-game/triplea/pull/598/files/1420ca8d2fed62a1fa0ecccd1c212a5efef61778..8eddebfb8bb8b20baf4e726edae241ea5aeedd3b#r59142323>
>